extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -1423,60 +1416,67 @@ public static String getIosUserAgent(@Nullable final Localization localization)
                 + ")";
     }
 
+    /**
+     * Returns a {@link Map} containing the required YouTube Music headers.
+     */
     @Nonnull
     public static Map<String, List<String>> getYoutubeMusicHeaders() {
-        final Map<String, List<String>> headers = new HashMap<>();
-        headers.put("X-YouTube-Client-Name", Collections.singletonList(youtubeMusicKey[1]));
-        headers.put("X-YouTube-Client-Version", Collections.singletonList(youtubeMusicKey[2]));
-        headers.put("Origin", Collections.singletonList("https://music.youtube.com"));
-        headers.put("Referer", Collections.singletonList("https://music.youtube.com"));
+        final var headers = new HashMap<>(getOriginReferrerHeaders(YOUTUBE_MUSIC_URL));
+        headers.putAll(getClientHeaders(youtubeMusicKey[1], youtubeMusicKey[2]));
         return headers;
     }
 
     /**
-     * Add required headers and cookies to an existing headers Map.
-     * @see #addClientInfoHeaders(Map)
-     * @see #addCookieHeader(Map)
+     * Returns a {@link Map} containing the required YouTube headers.
```

Comment:
```suggestion
     * Returns a {@link Map} containing the required YouTube headers, including the
     * <code>CONSENT</code> cookie to prevent redirects to <code>consent.youtube.com</code>
```